Dockerfile-csi-host-definer

```diff
@@ -0,0 +1,39 @@
+FROM registry.access.redhat.com/ubi8/python-38:1-75.1638364053 as builder
+
+RUN pip3 install --ignore-installed --default-timeout=100 --upgrade pip==21.2.4
+WORKDIR /tmp
+COPY controllers/servers/host_definer/requirements.txt .
```

Comment:
since the project will use the mediators that are not sufficient and we need all the controller requirements

Reply:
all of the controller requirements